src/main/app/case/CaseApi.ts

```diff
@@ -163,8 +176,18 @@ export const getCaseApi = (userDetails: UserDetails, logger: LoggerInstance): Ca
   );
 };
 
-interface GetCaseResponse {
-  id: string | number;
+interface CcdV1Response {
+  id: string;
   state: State;
   case_data: CaseData;
 }
+
+interface CcdV2Response {
+  id: string;
```

Comment:
can make this extend v1

Reply:
I don't think I can because V2 doesn't have case_data